### PR TITLE
feat: Add Freya to the list of Rust projects using Accesskit.

### PR DESCRIPTION
**Pull request by marc2332**

### CODE REVIEW

File: content/_index.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ The following Rust projects already integrate AccessKit:
 * [egui](https://github.com/emilk/egui)
 * [Slint](https://slint.dev/)
 * [Xilem](https://github.com/linebender/xilem)
+* [Freya](https://freyaui.dev/)
```
Comment (DataTriny) [inline content/_index.md:18]: I'd prefer to keep this list alphabetically sorted. We probably don't want to give the impression that AccessKit website authors favor some UI toolkits over others.

**Review by DataTriny — CHANGES_REQUESTED** [review]

File: content/_index.md (hunk 1 above)
Comment (marc2332) [reply content/_index.md:18]: ohh sure!

**Review by DataTriny — APPROVED** [review]

**Comment by DataTriny** [comment]
Thank you @marc2332, content is OK for me.

I'd probably delay merging this until https://github.com/marc2332/freya/issues/966 is resolved though, as anyone testing this would be very disappointed at the moment.

**Comment by DataTriny** [comment]
Merging, as promised. Thank you for the quick fix!

**Pull request merged**